godump.go

```diff
@@ -553,6 +565,10 @@ func (d *Dumper) printValue(w io.Writer, v reflect.Value, indent int, visited ma
 
 // asStringer checks if the value implements fmt.Stringer and returns its string representation.
 func (d *Dumper) asStringer(v reflect.Value) string {
+	if d.disableStringer {
+		return ""
+	}
```

Comment:
Do we really want to return blank? I would think we would at least want the following to return the raw "non-stringer" value from the struct ? I could be wrong here

```go
	if !val.CanInterface() {Add a comment on lines R573 to R574Add diff commentMarkdown input: edit mode selected.WritePreviewAdd a suggestionHeadingBoldItalicQuoteCodeLinkUnordered listNumbered listTask listMentionReferenceSaved repliesAdd FilesPaste, drop, or click to add filesCancelCommentStart a reviewReturn to code
		val = forceExported(val)
	}
```